pmm_qa/haproxy_setup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+
+while [ $# -gt 0 ]; do
+
+   if [[ $1 == *"--"* ]]; then
+        param="${1/--/}"
+        declare $param="$2"
+   fi
+
+  shift
+done
+
+if [ -z "$metrics_mode" ]
+then
+      export metrics_mode=push
+fi
+
+# Install the dependencies
+source ~/.bash_profile || true;
+apt-get update
+apt-get -y install wget curl git gnupg2 lsb-release
```

Comment:
Thanks Alex, since now we are focusing on the migration most files are from qa (https://github.com/percona/pmm-qa) that have been moved here, nothing has been changed here.
But as you suggest, yes we can make it uniform when these files are adjusted.